packages/mui/src/components/MuiThemeProvider/theme.ts

```diff
@@ -29,9 +29,18 @@ const fontFamily = [
   '"Noto Color Emoji"',
 ].join(',');
 
-const primaryColor = getEnvironment('REACT_APP_PRIMARY_COLOR', '#0dbd8b');
+const primaryColorlight = getEnvironment(
+  'REACT_APP_LIGHT_PRIMARY_COLOR',
+  '#007a61',
+);
+const primaryColorDark = getEnvironment(
+  'REACT_APP_DARK_PRIMARY_COLOR',
+  '#129a78',
+);
 const primaryColorHighContrast = '#075D53';
 const errorColor = '#ff5b55';
```

Comment:
I think this variable is no longer required and we change the baseTheme error color to errorColorLight.